rust/gui-client/src-tauri/Cargo.toml

```diff
@@ -43,7 +43,7 @@ tauri = { version = "1.6", features = [ "dialog", "shell-open-api", "system-tray
 tauri-runtime = "0.14.2"
 tauri-utils = "1.5.3"
 thiserror = { version = "1.0", default-features = false }
-tokio = { version = "1.36.0", features = ["time"] }
+tokio = { version = "1.36.0", features = ["signal", "time"] }
```

Comment:
I wanted graceful Ctrl+C for the debug subcommand. It's not used in the GUI itself, although there's no reason we couldn't.

Reply:
I think this is also why the Gateway doesn't respond to Ctrl-C? I think that causes the docker container to delay restart/shutdown

Reply:
I had thought it was because the `init` flag wasn't set in our Dockerfile, but I can't remember if that fixed it. It seemed to help for the Client.

I've definitely seen it before on other Docker containers, you have to catch Ctrl+C and also tell Docker to forward it.